suite-native/graph/src/reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { TimeFrameValues } from './types';
+
+interface AppGraphState {
+    selectedTimeFrame: TimeFrameValues;
+}
+
+const initialState: AppGraphState = {
+    selectedTimeFrame: 'day',
+};
+
+interface RootState {
```

Comment:
It should be in `store.ts`, right?

Reply:
This is `RootState` of this slice. Bad name I guess. I will rename it to `SliceState`

Reply:
https://github.com/trezor/trezor-suite/pull/5838/commits/8d03ac13bbdc855f35c585b400951d9764b552cb